backend/connections.ts

```diff
@@ -0,0 +1,66 @@
+import { Handler } from "aws-lambda";
+import { ApiGatewayManagementApi } from "aws-sdk";
+
+import Connections from "../common/infra/Connections";
```

Comment:
@douglasnaphas do you think that having this code here is 
1. weird
2. problematic

I am experimenting with the idea of having the tables infrastructure defined along with its use since if one changes the other is likely to need to change. 

in this case it looks like 

```
room-app/
+ backend
    + connect
        + // etc
+ common
    + infra // this name might be weird
        - Connections.ts
+ lib
    + // etc
// etc
```